portfolio/src/main/webapp/script.js

```diff
@@ -12,6 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+fetch('/data').then(response => response.json()).then((data) => {
```

Comment:
This code could benefit from a comment that describes what's going on. For now it looks like you're just logging it, but you can even state that in the comment, i.e.

```
// This code gets comment data from the /data backend path and updates the #display-comments element
// with comment data. TODO: Actually update the #display-comments element.
```

Not sure if this is what you intended to do with this code, but that'd just be an example. This makes it easier for people to work with your code even when you've left it in a partial state.

Reply:
got it